spec/models/manageiq/providers/openstack/cloud_manager/vm_spec.rb

```diff
@@ -57,8 +57,8 @@
         expect(vm.power_state).to eq 'migrating'
       end
 
-      it "checks evacuation is_available?" do
-        expect(vm.is_available?(:evacuate)).to eq true
+      it "checks vm supports evacuate" do
```

Comment:
while you touch this spec, it's good to rephrase it to something meaningful.
The `it` part should always read like a human description of the test. So in this case it would be 
`it "returns true for querying if the evacuate operation is supported"` 

if you run 
`rspec --format documentation spec/models/manageiq/providers/openstack/cloud_manager/vm_spec.rb`
it should produce a nice readable output